<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataArrayCreationWidget</class>
 <widget class="QWidget" name="DataArrayCreationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>55</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Property</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="dataContainerList"/>
   </item>
   <item>
    <widget class="QComboBox" name="attributeMatrixList"/>
   </item>
   <item>
    <widget class="QLineEdit" name="dataArrayName"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
